.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "external/stb"]
```

Comment:
One thought here:

If the main features being imported from stb are the header files "stb_image.h" and "stb_image_write.h", would it be more straightforward to include these two headers directly in MaterialXRender, in the same fashion that TinyEXR is currently included?  Although there may be benefits in including larger libraries through Git submodules in the future, this might not be a situation that warrants that degree of overhead.